app/controllers/containers_controller.rb

```diff
@@ -35,10 +35,10 @@ def show
   end
 
   def commit
-    Docker::Container.get(@container.uuid).commit(:author  => params[:commit][:author],
-                                                  :repo    => params[:commit][:repo],
-                                                  :tag     => params[:commit][:tag],
-                                                  :comment => params[:commit][:comment])
+    ForemanDocker::Docker.get_container(@container).commit(:author  => params[:commit][:author],
+                                                           :repo    => params[:commit][:repo],
```

Comment:
Line is too long. [94/80]<br>Unnecessary spacing detected.<br>Use the new Ruby 1.9 hash syntax.